.env

```diff
@@ -1,2 +1,3 @@
 PORT = 7000
-dbURI = mongodb+srv://newblog:newblog@cluster0.xtors.mongodb.net/?retryWrites=true&w=majority
\ No newline at end of file
+dbURI = mongodb+srv://newblog:newblog@cluster0.xtors.mongodb.net/?retryWrites=true&w=majority
+DBURI = mongodb://localhost:27017/newblog
```

Comment:
You should not commit your `.env` file.
It should be `NAME=VALUE`, not `NAME = VALUE`

Reply:
You can use '_' to separate words

Reply:
ok